src/site/content/en/blog/model-viewer/index.md

```diff
@@ -0,0 +1,160 @@
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - 3d
+  - model-viewer
+---
+
+Adding 3D models to a website is tricky. 3D models ideally will be shown in
+a viewer that can work responsively on all browsers including smartphones,
+desktop, or even the new head-mounted displays. The viewer should support
+progressive enhancement for performance and rendering quality. It should support
+use cases on all devices ranging from older, lower-powered smartphones to newer
+devices that support augmented reality. It should stay up to date with current
+technologies. It should be performant and accessible. However, building such a
+viewer requires specialty 3D programming skills, and can be a challenge for web
+developers that want to host their own models instead of using a third-party
+hosting service.
+
+<style>
+  #mv-demo {
+    background-color: #36b6fe;
+    max-height: 400px;
+    height: 400px;
+    width: 100%;
+  }
+</style>
+
+To help with that, the `<model-viewer>` web component, which just released
+[version 1.0](https://modelviewer.dev/), lets you declaratively add a 3D model
+to a web page, while hosting the model on your own site. The web component
+supports responsive design and use cases like augmented reality on some devices,
+and it includes features for accessibility, rendering quality, and
+interactivity.  The goal of the component is to enabble adding 3D models to your
+website without understanding the underlying technology and platforms.
+
+{% Aside %}
+We're always [updating and
+improving](https://github.com/google/model-viewer/releases)
+`<model-viewer>`. Check out the [`<model-viewer>`
+homepage](https://modelviewer.dev/) to explore what &lt;model-viewer&gt; 1.0 can do.
+{% endAside %}
+
+## What is a web component?
+
+A web component is a custom HTML element built from standard web platform
+features. A web component behaves for all intents and purposes like a standard
+element. It has a unique tag, it can have properties and methods, and it can
+fire and respond to events. In short, you don't need to know anything special to
+use any web component, much less `<model-viewer>`.
+
+In this article, I will show you things that are particular to `<model-viewer>`.
+If you're interested in learning more about web components
+[webcomponents.org](https://www.webcomponents.org/) is a good place to start.
+
+## What can &lt;model-viewer> do?
+
+I'll show you a few current capabilities of `<model-viewer>`. You'll get a great
+experience today, and `<model-viewer>` will get better over time as Google adds
+new features and improves rendering quality. The examples I'm provided are just
+to give you a sense of what it does. If you want to try them there are
+installation and usage instructions in [its GitHub
+repo](https://modelviewer.dev/).
+
+### Basic 3D models
+
+Embedding a 3D model is as simple as the markup below. By
+using glb files, we've ensured that this component will work on any major
```

Comment:
I think we need a link that explains what glb files are, because the intro of this post is saying "now you can use 3D models without being an expert" (i.e. we're not assuming 3D model expertise so our audience may not know what glb files are).